cmake/modules/AddSwift.cmake

```diff
@@ -160,8 +160,14 @@ function(_add_variant_link_flags
 
   if("${sdk}" STREQUAL "LINUX")
     list(APPEND result "-lpthread" "-ldl")
+    if("${arch}" MATCHES "^arm")
```

Comment:
It it sometimes one and sometimes the other.  I would prefer to hardcode the exact list here to make maintenance easier.  Many people need to modify this file, but very few can test on more rare platforms.